bindings/python/iree/runtime/function.py

```diff
@@ -381,11 +381,22 @@ def convert(inv: Invocation, vm_list: VmVariantList, vm_index: int, desc):
   return convert
 
 
+def _vm_to_pylist(inv: Invocation, vm_list: VmVariantList, vm_index: int, desc):
+  # The descriptor for a pylist is like:
+  #   ['pylist', element_type]
```

Comment:
'py_homogenous_list` (or whatever you name it)?

Reply:
thanks for catching that